### ISSUE TITLE
Stadia Maps example shows error message in the map

### ISSUE THREAD

**Issue by RalphLAtGitHub** [openlayers/openlayers#15212]
Error message is shown in map: "These basemap tiles will no longer be available ..." - see attached screenshot.
![Screenshot 2023-10-06 172715](https://github.com/openlayers/openlayers/assets/34791469/dd9043fb-5db1-4cf3-a9aa-b00db8db1fed)

**Comment by RalphLAtGitHub** [comment]
Yes, it also happens with the 8.1.0 version.
![Screenshot 2023-10-06 200612](https://github.com/openlayers/openlayers/assets/34791469/b64bec7f-e6a4-49a0-b765-b5846596ae6f)

**Comment by mike-000** [comment]
Are you running the example from https://openlayers.org/en/latest/examples/stamen.html or a copy on another site?  You will see that message if you are running on another site (or you are suppressing referrer) unless you use a Stadia API key, see https://docs.stadiamaps.com/authentication/

**Comment by RalphLAtGitHub** [comment]
Yes, I was runnig it yesterday from the original website. Today it surprisingly works without any error messages. If I switch to CodeSandbox using the EDIT button, the problem occurs again.
![Screenshot 2023-10-07 171544](https://github.com/openlayers/openlayers/assets/34791469/b41c55d6-4418-4bb3-8e3a-cee928838501)
